docs/reverse_proxy.md

```diff
@@ -98,10 +98,16 @@ example.com:8448 {
 
 ### Apache
 
+Enable `mod_proxy`, `mod_proxy_http`, `mod_proxy_balancer`, `mod_headers`
+and `mod_lbmethod_byrequests`.
```

Comment:
`mod_proxy_balancer` and `mod_lbmethod_byrequests` are likely only required if you require load balancing, which this example doesn't specify?